### ISSUE TITLE
Release of icons with android library

### ISSUE THREAD

**Issue by elax46** [iconify/iconify#289]
Hi, I wanted to know if you were planning to release the icons via library that can be implemented in Android applications. This question arises from the fact that I want to implement icons within wear OS but not having WebViews support they are not displayed. the only way is to implement a library that can allow them to be added

**Comment by cyberalien** [comment]
I don't know anything about Android development, so don't know what format is used, how to convert it, what part of SVG can be converted and so on.

**Comment by userquin** [comment]
@cyberalien I guess it is about creating vector assets using svg (using local file in the wizard below)

@elax46 this is something you need to build, iconify is about svg and html, maybe we can check how to build a kotlin snippet via jetpack compose api for svg: md3 has something similar for the md3 theme here https://m3.material.io/theme-builder#/custom (check export button int he top navbar)

There is a figma plugin here but not tested yet: https://www.figma.com/community/plugin/1034969338659738588/material-theme-builder

![imagen](https://github.com/iconify/iconify/assets/6311119/526ab452-6d68-4eeb-b643-c380bcbcff9e)

**Comment by cyberalien** [comment]
It should be automated, wizard cannot be used.

**Comment by userquin** [comment]
@cyberalien yeah, this is a wizard inside Android Studio (check screenshot below): with the new UI (jetpack compose), there are some api to build the new ui stuff without xml like on the export button with the md3 theme.

IIRC, the figma plugin should have an utility to export kotlin stuff for the md3 theme and so if we can add svg images there, the kotlin code should be also there. OFC, you need figma (I have no idea if the figma plugin will allow to add svg images, I think it should), otherwise we need to create the corresponding kotlin code...

![imagen](https://github.com/iconify/iconify/assets/6311119/5ec8e2ce-be91-4b54-bfa2-da92b22b0507)

**Comment by userquin** [comment]
The api can be found here:
- https://developer.android.com/jetpack/compose/resources#vector-assets
- https://developer.android.com/jetpack/compose/resources#animated-vector-drawables
- built-in md icons: https://developer.android.com/jetpack/compose/resources (we need to create this)

**Comment by cyberalien** [comment]
Nice. So it might be possible.

**Comment by elax46** [comment]
Yes, I know how Android Studio works and how to integrate files, I need to study better how to create a library that can be updated dynamically or automatically

**Comment by userquin** [comment]
There are a few icons in the compose library: https://material.io/blog/jetpack-compose-beta

We need to check the source code of those libraries: https://fonts.google.com/icons?selected=Material+Icons:fingerprint:&icon.platform=android
